content/blog/tina-is-in-beta.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Tina is in Beta
+date: '2021-07-27T20:00:00-04:00'
```

Comment:
Maybe we should change the date to today's date?

Reply:
Yup it's on the todo, still editing content